### PR TITLE
Hide "more" button on settings when not needed

### PR DESCRIPTION
**Pull request by samchan0221**
ref #1061

### CODE REVIEW

File: resources/authgear/templates/en/web/settings.html [hunk 1]
```diff
@@ -142,12 +144,14 @@ <h2 class="grid-area-title margin-0 primary-txt text-base font-semibold">
   {{ end }}
   {{ end }}
 
+  {{ if $show_more_button }}
   <section class="padding-20 row-sep">
     <a class="link action text-sm" href="/settings/identity">
       <i class="ti ti-chevrons-right"></i>
       {{ template "more-button-label" }}
```
Comment (louischan-oursky) [inline resources/authgear/templates/en/web/settings.html:151]: The label has to be updated. See https://github.com/authgear/authgear-server/issues/1061#issuecomment-890772930

Comment (louischan-oursky) [reply resources/authgear/templates/en/web/settings.html:151]: Ping

Comment (samchan0221) [reply resources/authgear/templates/en/web/settings.html:151]: Ok, I will update it

File: resources/authgear/templates/en/translation.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
   "disconnect-button-label": "Disconnect",
   "add-button-label": "Add",
   "remove-button-label": "Remove",
-  "more-button-label": "More",
+  "more-button-label": "More login methods",
```
Comment (louischan-oursky) [inline resources/authgear/templates/en/translation.json:71]: Do not change this label. Add a new key `more-login-methods-label`.
Also add this new key to `zh-HK` and `zh-TW` with `更多登入方法`

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

**Pull request merged**